src/index.js

```diff
@@ -81,6 +81,7 @@ const solrClient = new SolrClient({
   rows: 20,
   pageStrategy: "paginate",
   view: initialCamera,
+  filters: [{field: "-relation_target", value: "*"}],
```

Comment:
The only change we need is to add this attribute. The `solr-faceted-search-react` library doc doesn't mention it but the `filters` property has already been included in the library query setup.